RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate UserNotificationPreferencesView to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce


Go to `Profile`, then click on `Notifications` and will appear the `Notification Preferences`

## Screenshots

## Types of changes




## Checklist



## Further comments
